site_root/input/resources/examples/Practitioner-ig-001.json

```diff
@@ -0,0 +1,20 @@
+{
+  "resourceType": "Practitioner",
+  "id": "ig-001",
+  "meta": {
+    "profile": [
+      "http://hl7.org/fhir/StructureDefinition/Practitioner"
+    ]
+  },
+  "identifier": [
+    {
+      "system": "https://kf-api-dataservice.kidsfirstdrc.org/investigators?name=Christine%20Edry%20Seidman",
+      "value": "Christine Edry Seidman"
```

Comment:
Does Seidman use her middle name?

Reply:
Eh, no in the data service :) We can change this.

Reply:
Updated!